tests/common/type_coverage.h

```diff
@@ -2,7 +2,7 @@
 //
 //  SYCL 2020 Conformance Test Suite
 //
-// Provide common functions for type coverage
+//  Provide common functions for type coverage
```

Comment:
Minor: Is it a change made by clang-format? If so, I think it tries to align with the line 3, so I would remove one space there before SYCL instead.